components/ReplySearchPageLayout.js

```diff
@@ -144,10 +145,10 @@ function ReplySearchPageLayout() {
   const statsData = listStatData?.ListReplies || {};
 
   return (
-    <Box pt={2}>
-      <Box display="flex" justifyContent="space-between" flexWrap="wrap">
+    <>
+      <Tools>
         <TimeRange />
-      </Box>
+      </Tools>
 
       <Filters className={classes.filters}>
```

Comment:
We should remove style `classes.filters`, it's written in `Filters` component.

Reply:
Thanks! It's removed in the PR that remove `ReplySearchPageLayout.js`.